ckb-importer/src/main/java/com/hartwig/serve/ckb/datamodel/treatmentapproaches/RelevantTreatmentApproachesFactory.java

```diff
@@ -19,49 +19,58 @@ private RelevantTreatmentApproachesFactory() {
     }
 
     @NotNull
-    public static List<RelevantTreatmentApproaches> extractRelevantTreatmentApproaches(@NotNull CkbJsonDatabase ckbJsonDatabase,
+    public static List<DrugClassTreatmentApproach> extractDrugTreatmentApproaches(@NotNull CkbJsonDatabase ckbJsonDatabase,
```

Comment:
There is a bunch of duplication here that could be refactored out between the two public methods.